content/en/docs/release_notes/flint.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Flint release notes
+linkTitle: Flint
+description: 28 January 2022
+date: 2022-01-28
+Hide_readingtime: true
+---
+## Summary
+In this release, we introduced smart body encoding and `content-type` header selection to the OpenAPI flow-trigger. Additionally, added support for overriding
+host, basePath and schemes in the OpenAPI 2.0 and servers in OpenAPI 3.0 api-docs respectively when `config.overrides` are enabled.
+
+{{% releasenotes/upgrade %}}
+
+## Breaking changes
+* #7168: [@axway/api-builder-plugin-ft-oas](https://www.npmjs.com/package/@axway/plugin-ft-oas) is more strict when a response body is sent without a `content-type` header, and will now return a `500 Server Error` if the correct `content-type` cannot automatically be determined from the OpenAPI document.
+* #7195: [@axway/api-builder-plugin-ft-oas](https://www.npmjs.com/package/@axway/api-builder-plugin-ft-oas) now relies on a minimum version of [@axway/api-builder-runtime@4.79.0](https://www.npmjs.com/package/@axway/api-builder-runtime)
```

Comment:
Not sure if we should care about saying specific runtime version numbers here since it's easier to just say the release name.